tests/e2e/helpers.py

```diff
@@ -33,18 +29,3 @@ def delete_trello_board(key, token, board_id):
     response = requests.delete(url=f'https://api.trello.com/1/boards/{board_id}', params=params)
     if response.status_code != 200:
         raise Exception(f'Attempting to delete e2e test board returned status code: {response.status_code}')
-
-@pytest.fixture(scope='module')
-def driver():
-    with webdriver.Firefox(executable_path='./bin/geckodriver') as driver:
-        yield driver
-
-def test_task_journey(driver, app_with_temp_board):
-    driver.get('http://localhost:5000/')
-    text_box = WebDriverWait(driver, timeout=5).until(lambda d: d.find_element_by_id('field_name'))
-    text_box.send_keys('Test Todo')
-    submit_button = WebDriverWait(driver, timeout=5).until(lambda d: d.find_element_by_id('submit'))
-    submit_button.click()
-    time.sleep(5)
-    cards = WebDriverWait(driver, timeout=5).until(lambda d: d.find_elements_by_id('card_label'))
-    assert cards[len(cards) - 1].text.startswith('Test Todo')
```

Comment:
Extracted this driver-specific code to `test_chromium` and `test_firefox`. Left common code in `helpers`.

Reply:
It's fine to drop Firefox support if you don't use it, up to you